src/index.ts

```diff
@@ -182,7 +182,7 @@ async function runGlobs(options: Options, pluginsDefaultOptions: PluginsOptions,
         const filePath = filesPathsTargets[i];
         const fileRelativePath = fastRelativePath(rootPath, filePath);
         //TODO: Make sure the error is syntax-highlighted when possible
-        if (options.check || options.write) {
+        if (options.check || options.write || options.dump) {
```

Comment:
importantly , this changed

this is what makes errors now log out when you're in dump mode (the same as prettier)